website/docs/r/sql_server.html.markdown

```diff
@@ -80,7 +97,19 @@ The following attributes are exported:
 
 -> You can access the Principal ID via `${azurerm_sql_server.example.identity.0.principal_id}` and the Tenant ID via `${azurerm_sql_server.example.identity.0.tenant_id}`
 
-## Timeouts
+---
+
+A `blob_extended_auditing_policy` block supports the following:
+
+* `storage_account_access_key` - (Required) Specifies the identifier key of the auditing storage account.
+
+* `storage_endpoint` - (Required) Specifies the blob storage endpoint (e.g. https://MyAccount.blob.core.windows.net).
+
+* `is_storage_secondary_key_in_use` - (Optional) Specifies whether storageAccountAccessKey value is the storage's secondary key.
```

Comment:
```suggestion
* `is_storage_secondary_key_in_use` - (Optional) Specifies whether `storage_account_access_key` value is the storage's secondary key.
```

if this is the case, would it make sense to add a new `storage_account_secondary_access_key` that conflicts with `storage_account_access_key` and sets `is_storage_secondary_key_in_use` to true?